custom/icds_reports/utils/data_accessor.py

```diff
@@ -48,3 +50,47 @@ def get_program_summary_data_with_retrying(step, domain, config, now, include_te
             sleep(5)
             retry += 1
     return data
+
+
+# keeping cache timeout as 2 hours as this is going to be used
+# in some script/tool which might flood us with requests
+@icds_quickcache(['state_id', 'month'], timeout=120 * 60)
+def get_disha_api_v2_data(state_id, month):
+
+    data = DishaIndicatorViewV2.objects.filter(month=month,
+                                               state_id=state_id).first()
+
+    #not using lxml because using lxml when no xml manipulation needed is just overengineering
+    api_xml_response = """
+    <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
These SOAP tags are given by one of the members in GS I think who was given this requirement.

Reply:
👍

Reply:
Can you point to where these come from? This is not actually a SOAP api (we do not accept SOAP requests) so adding all that is at best confusing, it seems likely that something may have been lost in communication. The spec looks like they were at one point asking for a genuine SOAP API, and this may be a holdover from that? As is, it does not make much sense, and if they are fine with a non SOAP api, they should be fine with us not adding the soap tags.

Reply:
Yeah. I explicitly asked this. And was told to put these soap tags. Will pitch again for this.